cirq-google/cirq_google/calibration/workflow.py

```diff
@@ -842,25 +842,21 @@ def run_calibrations(
     elif isinstance(sampler, QuantumEngineSampler):
         engine = sampler.engine
         (processor_id,) = sampler._processor_ids
-        gate_set = sampler._gate_set
     else:
         engine = None
 
     if engine is not None:
         if processor_id is None:
-            raise ValueError('processor_id must be provided.')
-        if gate_set is None:
-            raise ValueError('gate_set must be provided.')
+            raise ValueError('processor_id must be provided.')  # coverage: ignore
```

Comment:
Did this file exist before coverage test was in place? Why does coverage check fail now but not before?

Reply:
I presume that's what happened; I didn't look into it too carefully.